src/rastervision/protos/label_store.proto

```diff
@@ -37,9 +39,17 @@ message ClassificationGeoJSONFile {
     optional Options options = 2;
 }
 
+message SegmentationRasterFile {
+    optional RasterSource src = 1;
```

Comment:
I think a comment for each of these fields would be helpful.

Reply:
Maybe comments would resolve my confusion, but more generally, I don't understand the notion of source and destination in this PR. Does source correspond to where the ground truth comes from and destination where the predictions go? If so, you should use `ground_truth_label_store` and for the ground truth and a separate `prediction_label_store` for the predictions to follow the convention in the rest of RV.

Reply:
Okay, will-do.

Reply:
> Maybe comments would resolve my confusion, but more generally, I don't understand the notion of source and destination in this PR. 

The nomenclature may be poorly chosoen.  Source things are given labels (e.g. the label rasters in potsdam).  In the case of rasters, the "destination" is where internally generated labeling information is sent; in the case of classes those are the classes used internally (e.g. "cars" being class of `1` instead of class `0xffff00` as they are in the given labels).